aws_lambda_powertools/logging/utils.py

```diff
@@ -75,6 +75,7 @@ def _find_registered_loggers(
 def _configure_logger(source_logger: Logger, logger: logging.Logger, level: Union[int, str]) -> None:
     logger.handlers = []
     logger.setLevel(level)
+    logger.propagate = False
```

Comment:
Wasn't aware of this side-effect. Will create test case for it and work to ensure there is no regression. Thanks for spotting this :-)

Reply:
Wasn't aware of this side-effect. Will create test case for it and work to ensure there is no regression. Thanks for spotting this :-)

Reply:
As I was running the tests locally and getting familiar with the code changes, I realized this wasn't true because we are filtering out child loggers by default -- copied config only applies to parent loggers so this edge case will never be hit.

I renamed the test covering keys being added after config is copied, so we can always come back to and know what this test is for.

> Repurposed test

```python
def test_copy_config_to_parent_loggers_only(stdout):
    # GIVEN Powertools Logger and Child Logger are initialized
    # and Powertools Logger config is copied over
    service = service_name()
    child = Logger(stream=stdout, service=service, child=True)
    parent = Logger(stream=stdout, service=service)
    utils.copy_config_to_registered_loggers(source_logger=parent)

    # WHEN either parent or child logger append keys
    child.append_keys(customer_id="value")
    parent.append_keys(user_id="value")
    parent.info("Logger message")
    child.info("Child logger message")

    # THEN both custom keys should be propagated bi-directionally in parent and child loggers
    # as child logger won't be touched when config is being copied
    parent_log, child_log = capture_multiple_logging_statements_output(stdout)
    assert "customer_id" in parent_log, child_log
    assert "user_id" in parent_log, child_log
    assert child.parent.name == service
```
> Where we ignore child loggers

```python
def _exclude_registered_loggers_filter(loggers: Set[str]) -> List[logging.Logger]:
    return [
        logging.getLogger(name) for name in logging.root.manager.loggerDict if "." not in name and name not in loggers
    ]
```